src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -182,6 +182,14 @@ public async Task<IActionResult> GetCommit(int buildId)
                 return NotFound();
             }
 
+            build = new Data.Models.Build();
```

Comment:
Intentional?

Reply:
I think you just need a refresh to look at the latest iteration. I was about the make the same comment on that whole block as well.

Reply:
Yup, shouldn't have commit it in the first place. I removed it afterwards :)